File: samples/issues-repro-samples/703-missing-methods/begin/Sample.WearOS/MainActivity.cs
using Android.Content.PM;

namespace Sample.WearOS;

[Activity(Label = "@string/app_name",
    MainLauncher = true,
    LaunchMode = LaunchMode.SingleInstance)]
public class MainActivity : Activity
{
    protected override void OnCreate(Bundle? savedInstanceState)
    {
        base.OnCreate(savedInstanceState);

        // Set our view from the "main" layout resource
        SetContentView(Resource.Layout.activity_main);
    }
}

File: samples/JetPack.Compose/start-original/SampleJetpackCompose/NativeApplicationWithNativeLib/app/src/main/java/com/example/nativeapplicationwithnativelib/MainActivity.kt
package com.example.nativeapplicationwithnativelib

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.example.mylibrary.ComposeActivity

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val intent = Intent(this, ComposeActivity::class.java)
        startActivity(intent)
    }
}